docs/src/reference/workflow-config-file.rst

```diff
@@ -513,17 +513,17 @@ alignment_reference
 ~~~~~~~~~~~~~~~~~~~
 
 -  type: string
--  description: Path to a FASTA-formatted sequence to use for alignment with ``nextalign``
+-  description: Path to a FASTA-formatted sequence to use for alignment with ``nextclade``
 -  default: ``defaults/reference_seq.fasta``
--  used in rules: ``align``, ``proximity_score`` (subsampling), ``build_align``, ``build_mutation_summary``
+-  used in rules: ``align``, ``proximity_score`` (subsampling)
 
 annotation
 ~~~~~~~~~~
 
 -  type: string
--  description: Path to a GFF-formatted annotation of gene coordinates (e.g., a “gene map”) for use by ``nextalign`` and mutation summaries.
+-  description: Path to a GFF-formatted annotation of gene coordinates (e.g., a “gene map”) for use by ``nextalign`` for codon-aware alignment.
```

Comment:
?? 

```suggestion
-  description: Path to a GFF-formatted annotation of gene coordinates (e.g., a “gene map”) for use by ``nextclade`` for codon-aware alignment.
```

Reply:
Ah, good catch! I missed this during rebasing.

Reply:
Fixed in [force-push](https://github.com/nextstrain/ncov/compare/c5ca8fee2c60a14786d51716ed90e8795bd536c9..f269e69b4921f93c37a0964accd4bdfc418ea1c5)